src/components/avatar/avatar.scss

```diff
@@ -0,0 +1,44 @@
+@use '../../design-tokens/colors.yml';
+@use '../../design-tokens/sizes.yml';
+@use '../../mixins/ratio-box';
+@use '../../mixins/theme';
+
+/**
+ * Themed properties
+ */
+
+@include theme.props() {
+  --theme-color-background-avatar: #{colors.$gray-light};
+}
+
+@include theme.props(dark) {
+  --theme-color-background-avatar: #{colors.$primary-brand-dark};
+}
+
+/**
+ * Containing element
+ *
+ * 1. Because avatars are a good thing to lazy-load, we want to show a
+ *    background color that will make that transition less jarring. This also
+ *    lets this function as a placeholder if desired.
+ * 2. Prevent this element from squashing or stretching within a flex container.
+ * 3. The most common use cases for avatars use a consistent width, so we set
+ *    that here.
+ */
+
+.c-avatar {
+  background: var(--theme-color-background-avatar); /* 1 */
+  border-radius: sizes.$radius-full;
+  flex: none; /* 2 */
+  width: sizes.$avatar-medium; /* 3 */
+  // Ratio box hack lets us support fixed _and_ percentage widths with ease
+  @include ratio-box.core-styles();
```

Comment:
This is absolutely a nit, but I usually encourage putting `@extend` and `@include` statements at the top of a CSS block. This makes it clear that they're inherited/injected, and that the following styles can be used to override.

Feel free to ignore this feedback if it doesn't align with the C4 house style.

Reply:
@spaceninja I don't have an opinion on this and can't remember why I'm putting it at the end!

Do you know if this is something that can be enforced in our linter?

(Side note: I've also noticed our linting/formatting isn't enforcing quote styles in Sass commands like `@use`.)

Reply:
Yeah, we should be able to use stylelint to do this. In my stylelint config, I use stylelint-order or enforce alphabetical order. I think that extends to putting most at-rules at the top, but I dont' remember off the top of my head.

Enforcing quote styles is also possible.

The C4 stylelint config is 99% just importing SUIT's config. And since we weren't using SCSS when I set that up, it doesn't include stylelint-scss like my personal config does. There's definitely room to improve our config, I just haven't prioritized it because it seemed like everyone else was happy and I wasn't writing much CSS. But I'm happy to put a bit more work into it.

You know I love my lint configs. 😀

Reply:
Thank you, @spaceninja, that's helpful info! I'll probably just manually move this for this PR but I think these tools are a good idea.